documentation/modules/ROOT/pages/integrations/tracing.adoc

```diff
@@ -0,0 +1,101 @@
+
+If the span context is not provided then the SMT will create a new span.
+In this case, Debezium operations together with metadata will be traced but will not be connected to business transaction traces to enable end-to-end testing.
+
+When this SMT is invoked with a message then it will:
+
+* extract parent span context if present in the message
+* create event `db-log-write` span context with the start timestamp set to the database log write timestamp
+* insert fields from `source` block into the span as *tags*
+* create processing `debezium-read` span as a child of `db-log-write` span with the start timestamp set to the processing time of the even
+* insert fields from envelope such as `op` into the processing span as *tags*
+* injects the processing span context into message headers
+
+=== Kafka Producer tracing
+
+Optionally it is possible to enable tracing at the Kafka producer level.
+If enabled then when the message is being written to the Kafka broker the producer will extract Debezium's processing span context from the Kafka message headers, create a new child span and record information about the write to the broker.
+Then it injects the new span into the message headers so a consumer of the message can restore the trace and resume end-to-end tracing.
+
+=== Configuration options
+
+[cols="65%a,>15%a,>20%"]
+|===
+|Configuration property
+|Type
+|Default
+
+|`tracing.span.context.field`::
+The name of the field containing span context. +
++
+_The sender must write the span context into the database column as a serialized instance of `java.util.Properties` with injected span context._
+|string
+|tracingspancontext
+
+|`tracing.operation.name`::
+The operation name representing the Debezium processing span. +
+|string
+|debezium-read
+
+|`tracing.with.context.field.only`::
+Only events that have serialized context field should be traced.
++
+_If set to `true` then tracing span will be created only for events with associated tracing span context field.
+If set to `false` then the tracing span is created for all incoming events regardless of having associated span context._
+|boolean
+|false
+
+|===
+
+== Outbox Extension
+
+Outbox extension provides the additional operations necessary for tracing context propagation out-of-the-box.
+It provides `tracingspancontext` field in an outbox table that is used as the vehicle to pass span context from service using the outbox extension to a Debezium connector.
+
+When an outbox even is emitted the extension will:
+
+* creates a new `outbox-write` span as a child of current active span
+* injects the span context into that `java.util.Properties` instance that is serialized into the `tracingspancontext` column
+* writes the record into the database
+
+== Event Router SMT
+
+Event Router acts as an Outbox Extension counterpart, executes the same steps as `ActivateTracingSpan` SMT, and is used instead of it.
```

Comment:
```suggestion
The Event Router SMT acts as an Outbox Extension counterpart, it executes the same steps as the `ActivateTracingSpan` SMT, and is used instead of it.
```